### PR TITLE
Changes related to the Game for sr 2020

### PR DESCRIPTION
**Pull request by TomWheal**
This PR contains commits relevant to the game for sr2020

DO NOT ADD THE ACTUAL RULES UNTIL KICKSTART

### CODE REVIEW

**Comment by RealOrangeOne** [comment]
Deploying the rules is part of https://github.com/srobo/tasks/issues/376, which I'll do on Saturday!

File: programming/sr/vision/markers.md [hunk 1]
```diff
@@ -18,9 +18,9 @@ Let's break that down:
 
 Details of the types and size of markers used in the SR2019 game can be found in the [rules](/docs/rules).
 
-You can download all of the markers as a single [ZIP file](/docs/resources/2019/sr-dev-markers-sr2019.zip)
-or individually from the [git repo](https://github.com/srobo/game-markers/tree/master/SR2019/dev).
-The *Arena* markers, due to their size, need to be printed on A3 paper.
+You can download all of the markers as a single [ZIP file](/docs/resources/2020/sr-dev-markers-sr2020.zip)
```
Comment (RealOrangeOne) [inline programming/sr/vision/markers.md]: I notice this points to a ZIP which doesn't exist yet?

Comment (kierdavis) [reply programming/sr/vision/markers.md]: Good timing, I just remembered this and came here to add it to this PR!

Comment (kierdavis) [reply programming/sr/vision/markers.md]: 652115f

File: rules/index.md [hunk 2]
```diff
@@ -6,7 +6,12 @@ title: 2020 Rules
 2020 Rules
 ==========
 
-The rules for this year's competition will be published during Kickstart.
+[<img class="left" src="{{ site.baseurl }}/resources/2020/rulebook.png" />]({{ site.baseurl }}/resources/2020/rulebook.pdf)
+The rules, regulations and specifications for the SR2020 competition can be
+found in the [rulebook]({{ site.baseurl }}/resources/2020/rulebook.pdf).
+
+The PDF may be subject to minor changes, so please ensure you're using the most up-to-date version.
+We will also let you know of any important changes by email.
```
Comment (PeterJCLaw) [inline rules/index.md]: Really? I think we normally post on the forums.

Comment (PeterJCLaw) [reply rules/index.md]: Hrm, looks like this was what we've said here in the past.

Comment (TomWheal) [reply rules/index.md]: I think thats a better way of announcing formal major changes as well imo

Comment (trickeydan) [reply rules/index.md]: Why not do both? Email can be the official way

**Pull request merged**